spec.html

```diff
@@ -1079,6 +1079,11 @@ <h1>The String Type</h1>
       </emu-note>
       <p>In this specification, the phrase "the <dfn id="string-concatenation">string-concatenation</dfn> of _A_, _B_, ..." (where each argument is a String value, a code unit, or a sequence of code units) denotes the String value whose sequence of code units is the concatenation of the code units (in order) of each of the arguments (in order).</p>
       <p>The phrase "the <dfn id="substring">substring</dfn> of _S_ from _inclusiveStart_ to _exclusiveEnd_" (where _S_ is a String value or a sequence of code units and _inclusiveStart_ and _exclusiveEnd_ are integers) denotes the String value consisting of the consecutive code units of _S_ beginning at index _inclusiveStart_ and ending immediately before index _exclusiveEnd_ (which is the empty String when _inclusiveStart_ = _exclusiveEnd_). If the "to" suffix is omitted, the length of _S_ is used as the value of _exclusiveEnd_.</p>
+      <p>
+        The phrase "<dfn id="ASCII-word-characters">ASCII word characters</dfn>" denotes the following String value, which consists solely of every letter and number in the Unicode Basic Latin block along with U+005F (LOW LINE):<br>
```

Comment:
Nit: maybe

```suggestion
        The phrase "<dfn id="ASCII-word-characters">the ASCII word characters string</dfn>" denotes the following String value, which consists solely of every letter and number in the Unicode Basic Latin block along with U+005F (LOW LINE):<br>
```

? Its usages read grammatically wrong to me otherwise.

Reply:
You're saying that you'd prefer «_the string-concatenation of [the ASCII word characters](https://tc39.es/ecma262/multipage/ecmascript-data-types-and-values.html#ASCII-word-characters) and **"-.!~\*'()"**_» rather than «_the string-concatenation of [ASCII word characters](https://tc39.es/ecma262/multipage/ecmascript-data-types-and-values.html#ASCII-word-characters) and **"-.!~\*'()"**_»? I prefer to leave off the definite article, but have no objection if there's consensus amongst editors to add it.

Reply:
I would prefer `the string-concatenation of the ASCII word characters string and "-.!~*'()"`, specifically, including both the definite article and the word "string".

Anyway I'll see what the other editors think.

Reply:
Other editors were in favor of the definite article and neutral-to-negative of "string" so let's do

```suggestion
        The phrase "<dfn id="ASCII-word-characters">the ASCII word characters</dfn>" denotes the following String value, which consists solely of every letter and number in the Unicode Basic Latin block along with U+005F (LOW LINE):<br>
```

and then yes, it would be used as e.g. `the string-concatenation of the ASCII word characters and "-.!~*'()"`

Reply:
@bakkot Done: 0bef1aa7906ff5f4774c24efb3820447769a1505